meta-store/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "quickwit-meta-store"
+version = "0.1.0"
+authors = ['Quickwit <hello@quickwit.io>']
+edition = "2018"
+
+[dependencies]
+anyhow = '1.0'
+async-trait = '0.1'
+thiserror = '1.0'
+
+quickwit-storage = { version = "0.1.0", path = "../storage" }
+
+[dependencies.rusoto_core]
```

Comment:
there should not be any dependency to rusoto.

This crate should only rely on the Storage abstraction instead of interfacing directly to S3.

Reply:
@fulmicoton 
Removed dependency to rusoto.